### ISSUE TITLE
Player Stats not changing. Once it changes , it resets back to the same value.

### ISSUE THREAD

**Issue by FireBlazR** [MAFINS/MenyooSP#165]

**Comment by ToxxicFuse** [comment]
Hi
Do you mean the abilities stats? (Stamina, Strength etc)

Could you also try changing the _Total Cash_ stat and see if that works? :)

**Comment by FireBlazR** [comment]
Yes, I mean the stamina , strength and stuff. I tried changing total cash, and it works fine.

**Comment by ToxxicFuse** [comment]
Okay thank you, I'll take a look at this

**Comment by FireBlazR** [comment]
Thanks a lot man...😊😊

**Comment by loriwood01** [comment]
Hi - I've been following this issue because I'm having the same problem. But I recently discovered a workaround - the stat changes stay if you change the status of the character you're not currently playing as.  It only reverts when you're trying to change the active character.

**Comment by jpcg7** [comment]
> Hi - I've been following this issue because I'm having the same problem. But I recently discovered a workaround - the stat changes stay if you change the stats of the character you're not currently playing as. It only reverts when you're trying to change the stats of the active character.

The problem is when you change back to the other character the stats reset to what they were

**Comment by loriwood01** [comment]
Yep, you're correct. Had the same thing happen after I wrote this.
